Commands/ImportReports.php

```diff
@@ -367,14 +367,18 @@ private function isValidDate($date)
             return \false;
         }
     }
-    public function checkIfCanProcess()
+    public function checkIfCanProcess($idSite)
     {
-        $nextAvailableAt = (int) Option::get(GoogleAnalyticsQueryService::DELAY_OPTION_NAME);
+        if (!$idSite) {
+            return true; // New import, so it should be allowed
```

Comment:
This method usually returns array and not boolean.
```suggestion
            return ['canProcess' => \true]; // New import, so it should be allowed
```